tableauserverclient/models/schedule_item.py

```diff
@@ -0,0 +1,239 @@
+import xml.etree.ElementTree as ET
+from .interval_item import *
+from .property_decorators import property_is_enum, property_not_nullable
+from .. import NAMESPACE
+
+
+class ScheduleItem(object):
+    class Type:
+        Extract = "Extract"
+        Subscription = "Subscription"
+
+    class ExecutionOrder:
+        Parallel = "Parallel"
+        Serial = "Serial"
+
+    class State:
+        Active = "Active"
+        Suspended = "Suspended"
+
+    def __init__(self, name, priority, schedule_type, execution_order, interval_item):
+        self._created_at = None
+        self._end_schedule_at = None
+        self._execution_order = None
+        self._frequency = None
+        self._id = None
+        self._name = None
+        self._next_run_at = None
+        self._priority = None
+        self._schedule_type = None
+        self._state = None
+        self._updated_at = None
+        self.interval_item = interval_item
+
+        # Invoke setter
+        self.execution_order = execution_order
+        self.name = name
+        self.priority = priority
+        self.schedule_type = schedule_type
+
+    @property
+    def created_at(self):
```

Comment:
We currently parse it as a string from the XML -- it's read only so I think that's fine